src/backends/tensorrt/tensorrtlib.cc

```diff
@@ -546,8 +547,13 @@ namespace dd
             this->_logger->info("found {} classes", _nclasses);
           }
 
-        if (_bbox && !this->_mlmodel._def.empty())
-          _top_k = findTopK(this->_mlmodel._def);
+        if (_bbox)
+          {
+            if (this->_mlmodel.is_onnx_source())
+              _top_k = onnx_proto::findTopK(this->_mlmodel._model);
+            else if (!this->_mlmodel._def.empty())
+              _top_k = findTopK(this->_mlmodel._def);
```

Comment:
Do we want this function to then be within a `caffe_proto::` scope ?

Reply:
I will sort this out